src/main/java/com/extendedclip/deluxemenus/menu/Menu.java

```diff
@@ -68,12 +68,13 @@ public Menu(String menuName, String menuTitle, Map<Integer, TreeMap<Integer, Men
   }
 
   public Menu(String menuName, String menuTitle, Map<Integer, TreeMap<Integer, MenuItem>> items,
-      int size) {
+      int size, List<String> args) {

```

Comment:
This new parameter isn't needed as this will only ever be set as `null`, which is already the default value for `args`.

Reply:
It is important as we cannot make an argument without a command otherwise

Reply:
Well, you can always use the other constructor.

This constructor has only 1 use and due to my other comment, the `args` value will always be set to null which is already the default value.

Reply:
The other constructor is not used without command in the code, it's why I added args to the default one. 
It's to make menu like that
![CleanShot 2023-12-30 at 23 15 59@2x](https://github.com/HelpChat/DeluxeMenus/assets/43671109/b3e40298-3fe9-437c-a7e9-ac26c1546b71)